catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/operation/impl/CreateRequestImpl.java

```diff
@@ -66,7 +66,10 @@ public CreateRequestImpl(List<Metacard> metacards, Map<String, Serializable> pro
   public CreateRequestImpl(
       List<Metacard> metacards, Map<String, Serializable> properties, Set<String> destinations) {
     super(properties);
-    this.metacards = metacards;
+    this.metacards = new HashSet<>();
+    for (Metacard metacard : metacards) {
```

Comment:
✏️ This isn't the most efficient way of doing this. Consider using the `addAll` method instead.